pages/windows/choco-pin.md

```diff
@@ -0,0 +1,20 @@
+# choco pin
+
+> Pin a package at a specific version with Chocolatey.
+> Pinned packages are skipped automatically when upgrading.
+
+- Display a list of pinned packages and their version:
```

Comment:
Maybe `and their versions` / `along with their versions`?

Reply:
True. :+1: I'll change this in the morning.